src/javase02/task02/Worker.java

```diff
@@ -0,0 +1,57 @@
+package javase02.task02;
+
+import java.util.Scanner;
+
+public class Worker {
+    String name;
+    int numberOfChancellery;
+    Chancellery[] chancelleries;
+    //Chancellery[] chancelleries = new Chancellery[numberOfChancellery];
+    static Scanner input = new Scanner(System.in);
+    private int calculatePriceOfChancellery(Worker obj){
```

Comment:
good point for creating unit test

Reply:
For now i dont know how to make them.

Reply:
https://junit.org/junit5/docs/current/user-guide/